content/en/docs/concepts/configuration/scheduling-framework.md

```diff
@@ -0,0 +1,230 @@
+scores from all plugins according to the configured plugin weights.
+
+### Normalize scoring
+
+These plugins are used to modify scores before the scheduler computes a final
+ranking of Nodes. A plugin that registers for this extension point will be
+called with the [scoring](#scoring) results from the same plugin. This is called
+once per plugin per scheduling cycle.
+
+For example, suppose a plugin `BlinkingLightScorer` ranks Nodes based on how
+many blinking lights they have.
+
+```go
+func ScoreNode(_ *v1.Pod, n *v1.Node) (int, error) {
+   return getBlinkingLightCount(n)
+}
+```
+
+However, the maximum count of blinking lights may be small compared to
+`NodeScoreMax`. To fix this, `BlinkingLightScorer` should also register for this
+extension point.
+
+```go
+func NormalizeScores(scores map[string]int) {
+   highest := 0
+   for _, score := range scores {
+      highest = max(highest, score)
+   }
+   for node, score := range scores {
+      scores[node] = score*NodeScoreMax/highest
+   }
+}
+```
+
+If any normalize-scoring plugin returns an error, the scheduling cycle is
+aborted.
+
+**Note:** Plugins wishing to perform "pre-reserve" work should use the
+normalize-scoring extension point.
+
+### Reserve
+
+This is an informational extension point. Plugins which maintain runtime state
+(aka "stateful plugins") should use this extension point to be notified by the
+scheduler when resources on a node are being reserved for a given Pod. This
+happens before the scheduler actually binds the pod to the Node, and it exists
+to prevent race conditions while the scheduler waits for the bind to succeed.
+
+This is the last step in a scheduling cycle. Once a pod is in the reserved
+state, it will either trigger [Un-reserve](#un-reserve) plugins (on failure) or
+[Post-bind](#post-bind) plugins (on success) at the end of the binding cycle.
+
+*Note: This concept used to be referred to as "assume".*
+
+### Permit
+
+These plugins are used to prevent or delay the binding of a Pod. A permit plugin
+can do one of three things.
+
+1.  **approve** \
+    Once all permit plugins approve a pod, it is sent for binding.
+
+1.  **deny** \
+    If any permit plugin denies a pod, it is returned to the scheduling queue.
+    This will trigger [Un-reserve](#un-reserve) plugins.
+
+1.  **wait** (with a timeout) \
+    If a permit plugin returns "wait", then the pod is kept in the permit phase
+    until a [plugin approves it](#frameworkhandle). If a timeout occurs, **wait**
+    becomes **deny** and the pod is returned to the scheduling queue, triggering
+    [un-reserve](#un-reserve) plugins.
+
+**Approving a pod binding**
+
+While any plugin can receive the list of reserved pods from the cache and
+approve them (see [`FrameworkHandle`](#frameworkhandle)) we expect only the permit
+plugins to approve binding of reserved Pods that are in "waiting" state. Once a
+pod is approved, it is sent to the pre-bind phase.
```

Comment:
nit: sometimes the "P" in pod is capitalized, sometimes it is not, perhaps be consistent with one or the other.

Reply:
Good point. In user docs we usually capitalize them. I updated the doc.